modules/Messenger/messenger_postProcess.php

```diff
@@ -232,7 +204,7 @@
 			//Role Categories
 			if (isActionAccessible($guid, $connection2, "/modules/Messenger/messenger_post.php", "New Message_role")) {
 				if ($_POST["roleCategory"]=="Y") {
```

Comment:
It looks like you're adding null coalesce to areas that are assigning values based on POST variables, which is certainly the most common ways POST variables are used. However, we can see that $_POST["roleCategory"] is being accessed directly, which will also cause an error if this value is not assigned. These are best not overlooked. 

One way to handle this is to create a variable with the null coalesce first, if it does not already exist, then check the value of the variable.

```suggestion
				$roleCategory = $_POST["roleCategory"] ?? 'N';
				if ($roleCategory == "Y") {
```

Can you look through the code for areas like this that you could improve? Thanks!